.github/workflows/helm.yaml

```diff
@@ -0,0 +1,115 @@
+name: helm
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch:
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+
+      - name: Update stackstorm-ha chart dependencies
+        run: |
+          set -x
+          helm dependency update
+
+      - name: Helm lint
+        run: |
+          helm lint
+
+      - name: Cache community
+        id: cache-community
+        uses: actions/cache@v3
+        with:
+          path: community
+          key: ${{ runner.os }}-community-${{ hashFiles('conf/**', 'templates/**', 'Chart.yaml', 'values.yaml') }}
+
+      - name: Helm template
+        if: steps.cache-community.outputs.cache-hit != 'true'
+        shell: bash
+        run: |
+          helm template --output-dir community .
+
+  k8s-lint:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    needs: [helm-lint]
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+
+      - name: Cache community
+        id: cache-community
+        uses: actions/cache@v3
+        with:
+          path: community
+          key: ${{ runner.os }}-community-${{ hashFiles('conf/**', 'templates/**', 'Chart.yaml', 'values.yaml') }}
+
+      - name: Kubernetes kubeval lint
+        uses: instrumenta/kubeval-action@master
+        with:
+          files: community
+
+  helm-e2e:
```

Comment:
I'm concerned that this job to 15 min. In CircleCI it takes around 6-7 minutes. Do you know what makes the difference?

Reply:
Hrm, the [most recent run](https://github.com/StackStorm/stackstorm-k8s/actions/runs/3735681553/jobs/6339213088) was just a handful of seconds.